spring-cloud-sleuth-core/src/main/java/org/springframework/cloud/sleuth/Span.java

```diff
@@ -127,4 +130,26 @@
 	 * Will never be null.
 	 */
 	List<Log> logs();
+
+
+	/**
+	 * Class used for conversions of long ids to their String representation
+	 */
+	class Converter {
```

Comment:
I can't use `Span.toHex` etc. since static methods in interface are allowed in JDK8 :)